Define which internal dependencies should be built.
I have attempted to build them with EB and link to them, but with no luck.
Author: Davide Vanzo (Vanderbilt University)
diff -ru fsl.orig/extras/build fsl/extras/build
--- fsl.orig/extras/build	2019-08-08 15:05:57.524644463 -0500
+++ fsl/extras/build	2019-09-06 10:48:43.244489983 -0500
@@ -106,6 +106,8 @@
 fi
 PROJECTS="${PROJECTS} libgd libgdc libprob libcprob newmat cprob newran fftw"
 PROJECTS="${PROJECTS} boost libxml2-2.9.2 libxml++-2.34.0 libsqlite libnlopt ../include/armawrap/dummy_newmat"
+# For EasyBuild
+PROJECTS="libgdc libprob libcprob cprob newran ../include/armawrap/dummy_newmat"
 for projname in $PROJECTS; do
     if [ -d $FSLESRCDIR/$projname ] ; then
        buildIt $FSLESRCDIR $projname 1 
